### ISSUE TITLE
Plane transforms to car on touch-down

### ISSUE THREAD

**Issue by TwinFan** [TwinFan/LiveTraffic#178]
**Describe the bug**
Planes (which are displayed with default type after no a/c type could be derived) can transform back to a car (ZZZC) shortly before or on touch-down

**To Reproduce**
Need to force „no a/c information“, switching off OpenSky Master channel should do
Watch planes landing

**Expected behavior**
A plane should stay a plane.
Ie. Transformation from car to plane upon take off is expected, back-transformation from plane to car upon touch-down is no good.

**Screenshots**

**Technical Info:**
 - OS: Any
 - X-Plane version: any
 - LiveTraffic version: introduced with v1.50

**Log.txt**

**Additional Info**
Reported on the [support forum](https://forums.x-plane.org/index.php?/forums/topic/207193-livetraffic-and-vulkanmetal/&do=findComment&comment=1907657)

**Comment by voipmeister** [comment]
I had this issue just now, made a screenshot for reference.
![issue-178](https://user-images.githubusercontent.com/12763807/80528548-66bd4080-8996-11ea-9f29-970f2d04894b.JPG)

**Comment by voipmeister** [comment]
Using LT 2.01.200503, just saw SEJ7010 flown by VT-SFB transform to a follow me car, just before or around the flare phase.

![image](https://user-images.githubusercontent.com/12763807/80922855-09533600-8d80-11ea-9cb9-9d97214a7e7f.png)

VT-SFB doesn't exist in the OpenSky database. I PM'd you my log.txt.

**Comment by TwinFan** [comment]
Honestly...I had no time to test the actual situation. I thought my coding would be bullet-proof...apparently it isn't.

The history of plane 800CF5 as per voipmeister's log:
```
0:27:35.933 LiveTraffic 03-May 18:41:550.5619Z DEBUG LTChannel.cpp:485/FetchAllData: OpenSky Masterdata Online: HTTP response is not OK but 404 for https://opensky-network.org/api/metadata/aircraft/icao/800CF5
...
0:27:36.462 LiveTraffic 03-May 18:41:560.0897Z DEBUG LTChannel.cpp:485/FetchAllData: OpenSky Masterdata Online: HTTP response is not OK but 404 for https://opensky-network.org/api/routes?callsign=SEJ7010
...
0:28:59.498 LiveTraffic 03-May 18:43:190.1116Z WARN  LTFlightData.cpp:2319/DetermineAcModel: Tracking data for '800CF5' (man '', mdl '') lacks ICAO a/c type code, will be rendered with standard a/c A320
0:28:59.498 LiveTraffic/XPMP2 03-May 18:44:55Z INFO  CSLModels.cpp:944/CSLFindMatch:  MATCH INPUT: Type=A320 (WTC=M,Class=L2J,Related=47), Airline=SEJ, Livery=
0:28:59.498 LiveTraffic/XPMP2 03-May 18:44:55Z INFO  CSLModels.cpp:1039/CSLFindMatch: MATCH FOUND: Type=A320 (WTC=M,Class=L2J,Related=47), Airline=SBI, Livery= / Quality = 12 -> BB_Airbus A320_SBI
0:28:59.498 LiveTraffic 03-May 18:43:190.1121Z INFO  LTAircraft.cpp:1243/LTAircraft: Added aircraft 800CF5 (A320 SEJ7010), operator '', a/c model 'BB_Airbus A320_SBI', flight model [MediumAirbus], bearing 69, distance 4.1nm, from channel OpenSky Live Online
...
0:30:30.611 LiveTraffic 03-May 18:44:500.2317Z INFO  LTAircraft.cpp:2829/ChangeModel: Changed CSL model for aircraft 800CF5 (ZZZC SEJ7010), operator '': a/c model now 'XCSL_CARS CARS_FollowMe'
...
0:33:39.466 LiveTraffic 03-May 18:47:590.0775Z INFO  LTAircraft.cpp:1259/~LTAircraft: Removed aircraft 800CF5 (ZZZC SEJ7010)
```

**Comment by TwinFan** [comment]
Why is `mdl` empty??? It should actually read `[?]` with v2.01! (That was part of the change to see no cars when there is no data.)

That makes me afraid that the binary didn't include the LiveTraffic part of the changes...id _does_ have the XPMP2 changes though as can be seen be some changes to logging messages (like seeing the actual model in the `MATCH FOUND` line. Also, all screenshots now show the last segment of the path to a CSL Model...but also that is primarily a XPMP2-side change.

**Comment by TwinFan** [comment]
Published v2.02, now (hopefully ;-) ) including the code for Windows, too.

**Comment by voipmeister** [comment]
Working as expected on Windows.

![image](https://user-images.githubusercontent.com/12763807/80940735-f45abf00-8de0-11ea-9319-afbb133f2d43.png)
